docs/DeveloperGuide.adoc

```diff
@@ -138,7 +138,7 @@ The `LogicManager` contains an `AnakinModel`, an `AnakinParser`, and a `CommandH
 When a string input by the user is parsed by `AnakinParser`, it creates a new `AnakinCommand` of the appropriate type with the relevant arguments (based on parser tokens). Each `AnakinCommand` has its own Class. `LogicManager` will call `execute` on the command object. If successful, `LogicManager` will modify `Model` accordingly. Regardless of success, `LogicManager` will update `CommandHistory` with the `CommandResult`.
 
 
-
+// tag::modelcomponent[]
```

Comment:
Just curious, why are we opting to comment this line out instead of deleting it straight away?

Reply:
It's for tagging purpose (other files can link to this part of developerguide)